current_spring2026/submit.sh

```
@@ -0,0 +1,18 @@
+#$ -N embed-fulltext
+#$ -j y
+#$ -o embed-fulltext.log
+#$ -l gpus=1
+#$ -l gpu_memory=16G
+#$ -l gpu_c=7.5
+#$ -pe omp 8
+
+# =============================================================================
+# Load config (universal params + task list)
+# =============================================================================
+source /etc/profile.d/modules.sh
```

Shellcheck (0.11.0)

[info] Not following: /etc/profile.d/modules.sh was not specified as input (see shellcheck -x).

(SC1091)